server/resources/resource_management_test.go

```diff
@@ -1160,6 +1162,17 @@ var _ = Describe("Resource manager", func() {
 				"test_integer": 1,
 				"test_bool":    false,
 			}
+			adminNetworkData = map[string]interface{}{
```

Comment:
these variables are used only in a single test case. could we move them to `BeforeEach` in `Context("As an admin"`?